spec/integration/appointment_spec.rb

```diff
@@ -0,0 +1,192 @@
+require 'swagger_helper'
+
+describe 'Appointments API' do
```

Comment:
Please add at least a user and a doctor before the tests so that we can run it test in swagger or on RSpec

Something like this should work:
```ruby
before do
    User.create(first_name: 'Test1', last_name: 'User', email: 'test@example.com', password: 'password123',
                date_of_birth: '10-01-2000', password_confirmation: 'password123')
end
```